website/content/docs/operators/configuration/config-intro.md

```diff
@@ -1,17 +1,18 @@
 ---
-title: Intro to Heron Configuration
+title: Intro to Heron Internal Configuration
 ---
 
-Heron can be configured at two levels:
+Heron internal configuration can be configured at two levels:
```

Comment:
I'm not sure how others feel about this, but it might be better to call this doc "Intro to Heron Cluster Configuration." "Internal" is just a bit too vague for my tastes. Then, the first few sentences could be something like this:

"Heron clusters can be configured at two different levels:

1. **The system level** --- System-level configurations apply to the whole Heron cluster rather than to any specific topology (e.g. logging configs).
2. **The component level** --- Component-level configurations enable you to establish default configurations for different topology components.

Neither system- nor component-level configurations can be overridden by topology developers."